BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Microsoft.ApplicationInsights.Tests.csproj

```diff
@@ -22,6 +22,7 @@
     <PackageReference Include="Moq" Version="4.10.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="2.2.3" />
     <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
+	<PackageReference Include="xunit" Version="2.4.1" />
```

Comment:
dont recollect the history, but the base sdk avoided xunit probably for a reason! (though i dont know if its still valid).
It'd be good to stick with existing test framework for now (unless you really need to use xunit)